examples/telemetry_channel_metrics.lua

```diff
@@ -0,0 +1,73 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Monitor telemetry traffic by channel
+
+*Example Heka Configuration*
+
+.. code-block:: ini
+
+    [ByChannelHour]
+    type = "SandboxFilter"
+    ticker_interval = 10
+    preserve_data = false
+    message_matcher = "Logger == 'telemetry'"
+
+        [ByChannelHour.config]
+        # Modify to adjust the aggregation
+        rows = 1440
+        sec_per_row = 3600
+
+--]]
+require "circular_buffer"
+
+local rows = read_config("rows") or 1440
+local sec_per_row = read_config("sec_per_row") or 60
+
+channel_names = {"nightly", "aurora", "beta", "release", "other"}
```

Comment:
The fx module has these defined, example use: https://github.com/mozilla-services/data-pipeline/blob/master/heka/sandbox/filters/count_by_normalized_channel.lua#L29